### ISSUE TITLE
Scrolling stops working in WhatsApp

### ISSUE THREAD

**Issue by joshuakaufman** [Caldis/Mos#668]
**Describe the bug | 问题描述**
When using WhatsApp, scrolling sometimes stops working altogether. Clicking resolves the issue and scrolling resumes. I'm using a Logitech MX Vertical mouse.

**To Reproduce | 如何重现**
Select a chat or the message list in WhatsApp and scroll. It doesn't happen consistently so you may need to try several chats.

**Expected behavior | 期望结果**
Smooth scrolling as usual.

**Screenshots | 相关截图**
None available.

**System Info | 系统信息:**
 - Mouse: Logitech MX Vertical
 - OS: macOS 14.6.1
 - Browser: N/A
 - Version: 3.4.1
 -  Application: WhatsApp

**Additional context | 额外说明**

**Comment by s1davide** [comment]
You are right that it does not work but in the new WhatsApp application, the scroll gets stuck and does not move in the chat section (right side).
![image](https://github.com/user-attachments/assets/33afa6c7-ed92-40e6-b1e6-fcb0681049ab)

**Comment by Jeromekem** [comment]
Also having this same issue !

**Comment by jwchong** [comment]
They stop updating the app for 2 years. Hope someone can fix this.

**Comment by hrvstr** [comment]
Somebody compiled a build a while ago for a similar problem. It might be worth a try.

https://github.com/Caldis/Mos/issues/410#issuecomment-1591095255

**Comment by jwchong** [comment]
I have bought https://macmousefix.com for USD3 (lifetime support) and no need to wait for MOS. MOS has stopped updating since 2 years ago. New bugs will be getting more and more. Thank you.

**Comment by s1davide** [comment]
> I have bought https://macmousefix.com for USD3 (lifetime support) and no need to wait for MOS. MOS has stopped updating since 2 years ago. New bugs will be getting more and more. Thank you.

It's not the same, Mos is much softer and more natural than any other app, I've already tried them all.

**Comment by jwchong** [comment]
I also hope to use MOS but no update for 2 years means they will not be updating it anymore. It will be incompatible with more app soon in the future. You will stop using it one day anyway.

**Comment by vitormiura** [comment]
same issue here

**Comment by vidurasilva84** [comment]
Same issue on Windows 11 x64.

**Comment by yaxhcodes** [comment]
Please someone fix this.

**Comment by hrvstr** [comment]
It looks like it has already been fixed, but there is no release. Another user compiled the changes and supplied a binary.

https://github.com/Caldis/Mos/issues/410#issuecomment-1591095255

**Comment by Jeromekem** [comment]
It's actually fixed . It was an issue with Catalyst. I attached the file again for convenience, it's somewhere in another github issue. 
[Mos.-.Fixed.Catalyst.Scrolling.Whatsapp.Messages.Universal.zip](https://github.com/user-attachments/files/17757994/Mos.-.Fixed.Catalyst.Scrolling.Whatsapp.Messages.Universal.zip)

**Comment by Caldis** [comment]
fixed at https://github.com/Caldis/Mos/releases/tag/3.5.0
